vita3k/host/include/host/state.h

```diff
@@ -67,6 +67,7 @@ struct HostState {
     uint32_t ms_per_frame = 0;
     bool should_update_window_title = false;
     WindowPtr window;
+    renderer::Backend backend_renderer;
```

Comment:
You can access the backend_renderer field via `renderer->backend`. Having a copy in host is kinda unnecessary and opens up the possibility of getting out of sync.

Reply:
i can't, use it causing crash, bercause it said i can't initialise value by app_init

Reply:
Keep `renderer->backend` the same for the entire execution. Don't change it- the only place that should change it is `renderer::init`.

Reply:
yes i know, but i canb't using it is my problem, it is impossible affect for setting anhd for init, both causing crash